bubblewrap.c

```diff
@@ -1869,7 +1869,7 @@ main (int    argc,
   acquire_privs ();
 
   /* Never gain any more privs during exec */
-  if (prctl (PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0) < 0)
+  if (is_privileged && prctl (PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0) < 0)
```

Comment:
shouldn't this be still safe with user namespaces as root setuid is mapped to the unknown user anyway?

Is it fine to move it after we write the uid/gid map (which is done in the parent process) and use it for the sandbox process?